docs/partners/integration_guide.md

```diff
@@ -42,3 +49,4 @@ Implications:
 - Loss of TVL attributed if users transfer the vault tokens
```

Comment:
Above it says partners deploy a routing contract for each vault then you say you can use multiple vaults per contract?

Reply:
well it's both according to docs: "A single routing contract can handle multiple vaults, but can also be deployed on a per vault basis"